helm_deploy/values-dev.yaml

```diff
@@ -20,3 +20,5 @@ ingress:
 
 env:
   JAVA_OPTS: "-Xmx1024m"
```

Comment:
looks far too big a value for memory - I would have thought 256 would be good starting point